### ISSUE TITLE
Cypress Tests Currently Fail

### ISSUE THREAD

**Issue by nbibler** [HaulinOats/todo-app#3]
Likely after the routing and design changes (perhaps the addition of a home page vs a todo listing), the Cypress tests no longer pass.

**Comment by HaulinOats** [comment]
Yeah, makes sense. Shouldn't have been so eager to push my code before crossing my T's first. This is one of those things where I'll need to give myself a checklist to make sure I don't miss anything.
